odbc_fdw.c

```diff
@@ -421,6 +476,15 @@ odbcGetOptions(Oid foreigntableid, char **svr_dsn, char **svr_database, char **s
     if (!*svr_dsn)
         *svr_dsn = NULL;
 
+    if (!*svr_driver)
+        *svr_driver = NULL;
+
+    if (!*svr_host)
+        *svr_host = NULL;
+
+    if (!*svr_port)
+        *svr_port = NULL;
```

Comment:
Mmm, when I first show that I thought it was handling the case of a empty string "" (which in C is just a 0 byte) and normalizing it to a `NULL` pointer, but now I see it's wrong, I think it should be:

```
if (!*svr_dsn)
         svr_dsn = NULL;
```

Or, if empty strings are preferred to NULL pointers: 

```
if (!svr_dsn)
         *svr_dsn = 0;
```

But given the checks elsewhere in the code: `if (svr_dsn)` it should be the first, so I'll fix it.